Mine Tune added

NkStevo opened the pull request:
Added a new Tune to the game which spawns a mine behind the player and
detonates if the enemy gets within its trigger. The mine does not
detonate when the owner passes through its trigger. The mine also
disappears after 4 seconds if it is not detonated so the owner cannot
litter the ground with mines.

TODO: Color of the mines needs to be dependent on the color of the owner
so that it is apparent who the mine belongs to.

NkStevo committed:
Mine Tune added

Added a new Tune to the game which spawns a mine behind the player and
detonates if the enemy gets within its trigger. The mine does not
detonate when the owner passes through its trigger. The mine also
disappears after 4 seconds if it is not detonated so the owner cannot
litter the ground with mines.

TODO: Color of the mines needs to be dependent on the color of the owner
so that it is apparent who the mine belongs to.

NkStevo committed:
Merge remote-tracking branch 'refs/remotes/kkiniaes/master'

# Conflicts:
#	Assets/Scene/Test.unity
#	Assets/Standard Assets/Prototyping/Models/Materials.meta
#	Assets/Standard Assets/Prototyping/Models/Materials/NavyGrid.mat.meta
#	ProjectSettings/ProjectSettings.asset

NkStevo commented:
Fixed the conflicts. Should be okay to merge.

JonathanHunter commented on ProjectSettings/ProjectSettings.asset:
```diff
@@ -0,0 +1,513 @@
+%YAML 1.1
```
Remove this file from your commit. It shouldn't be in the repo.

NkStevo replied:
Removed it. Let me know if there's anything else.

NkStevo committed:
Removed ProjectSettings.asset

Must be removed from commit

NkStevo committed:
Deleted unneccesary code from Mine.cs

The code deleted was unnecessary. The mine's particle system does not
need to be dealt with.
